### PR TITLE
Announce Broadcast

### PR DESCRIPTION
**Pull request by lislis**

### CODE REVIEW

File: _posts/2015-10-23-Transmitting-broadcasts.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+permalink: 2015-10-23-transmitting-broadcasts
+---
+
+
+<figure class="right small no-border">
+  ![transmitting](/images/2015-10-23-broadcast-tower.gif)  
+</figure>
+
+In case you haven't noticed it already, our broadcasts appear in a shiny new home.  
+Now located in the main top bar next to our logo, the little broadcast tower will notify you of any news we want you to know about.
```
Comment (BanzaiMan) [inline _posts/2015-10-23-Transmitting-broadcasts.md]: Perhaps we can provide a screen shot here, to give a graphical context.

File: _posts/2015-10-23-Transmitting-broadcasts.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+created_at: Thu Oct 23 2015 12:00:00 CEST
+permalink: 2015-10-23-transmitting-broadcasts
+---
+
+
+<figure class="right small no-border">
+  ![transmitting](/images/2015-10-23-broadcast-tower.gif)  
+</figure>
+
+In case you haven't noticed it already, our broadcasts appear in a shiny new home.
```
Comment (BanzaiMan) [inline _posts/2015-10-23-Transmitting-broadcasts.md]: I'm not a big fan of "In case you haven't noticed". The sentence reads fine without it.

**Comment by BanzaiMan** [comment]
:confetti_ball:

File: _posts/2015-10-23-Transmitting-broadcasts.md (hunk 1 above)
Comment (lislis) [inline _posts/2015-10-23-Transmitting-broadcasts.md]: I wanted to, too. But then had difficulties deciding on the display window. Only the logo and navigation seemed not enough but more context let the icon look very small. But I'll have a think again

File: _posts/2015-10-23-Transmitting-broadcasts.md (hunk 2 above)
Comment (lislis) [inline _posts/2015-10-23-Transmitting-broadcasts.md]: ok

**Comment by BanzaiMan** [comment]
:shipit:

**Comment by saltinejustine** [comment]
OMG OMG OMG! YAY!

**Pull request merged**